chainer/functions/noise/dropout.py

```diff
@@ -133,11 +156,22 @@ def dropout(x, ratio=.5, **kwargs):
         True
 
     """
-    argument.check_unexpected_kwargs(
-        kwargs, train='train argument is not supported anymore. '
-        'Use chainer.using_config')
-    argument.assert_kwargs_empty(kwargs)
+    mask = None
+    return_mask = False
+    if kwargs:
+        mask, return_mask = argument.parse_kwargs(
+            kwargs, ('mask', mask), ('return_mask', return_mask),
+            train='train argument is not supported anymore. '
+                  'Use chainer.using_config')
```

Comment:
For the record, manually resolved conflicts here.

Reply:
Could you check Jenkins errors here? Unexpected kwargs in `argument.parse_kwargs` is not backported. (#4880)